bin/resources/GameIndex.yaml

```diff
@@ -32278,7 +32278,7 @@ SLPM-62124:
   name-en: "Ready 2 Rumble Boxing - Round 2"
   region: "NTSC-J"
 SLPM-62125:
-  name: Gauntlet Dark Legacy
+  name: GAUNTLET DARK LEGACY
```

Comment:
Are you sure about this? That doesn't feel right.

Reply:
It doesn't look to be all caps on redump, but there is a hyphen/separator for the subtitle http://redump.org/disc/72339/

Reply:
This change is based on the PS3 BC compatibility list. But not a strong opinion as there is no consensus

Reply:
I think I'd believe Redump in this instance tbh xD Seems weird to capitalise that. I can understand it in KOF MAXIMUM IMPACT, but not for Gauntlet, pretty sure the other regions don't do it either.